ports/nrf/common-hal/audiobusio/I2SOut.c

```diff
@@ -283,6 +283,9 @@ void common_hal_audiobusio_i2sout_play(audiobusio_i2sout_obj_t* self,
     i2s_buffer_fill(self);
 
     NRF_I2S->RXTXD.MAXCNT = self->buffer_length / 4;
+    // Turn on the interrupt to the NVIC but not within the NVIC itself. This will wake the CPU and
+    // keep it awake until it is serviced without triggering an interrupt handler.
+    NRF_I2S->INTENSET = I2S_INTENSET_TXPTRUPD_Msk;
```

Comment:
At some point we will want to switch to having interrupts directly register the background work that is needed, instead of consulting a bunch of flags in run_background_tasks_if_tick.  I don't think it will, but just to make sure: does this change present any additional difficulty in doing that?

Reply:
No, I don't think so. I'd be fine for an actual interrupt handler to run and then set a flag. The important bit is that an interrupt occurs (otherwise it will stay asleep.)